great-idea/index.html

```diff
@@ -15,9 +15,72 @@
 </head>
 
   <body>
-
     <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <header>
+    <nav>
+      <a href="./services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+      <img class='logo' src='./img/logo.png' alt='company logo' />
+    </nav>
+  </header> 
+
+    <section class="one">
```

Comment:
I would rename this class to something more descriptive like tagline or call to action.